packages/core/useWheelScrollDirection/index.ts

```diff
@@ -0,0 +1,55 @@
+import { unref } from 'vue-demi'
+import type { MaybeRef } from '@vueuse/shared'
+import { useEventListener } from '@vueuse/core'
+
+export interface UseWheelScrollDirectionOptions {
+  direction?: MaybeRef<'horizontal' | 'vertical' | 'auto' | undefined>
+  scrollDeltaRatio?: MaybeRef<number>
+}
+
+/**
+ * Restrict wheel scroll direction.
+ *
+ * @param element
+ * @param options - UseWheelScrollDirectionOptions
+ * @see https://vueuse.org/useWheelScrollDirection
+ */
+export function useWheelScrollDirection(element: MaybeRef<HTMLElement | SVGElement | Window | Document | null | undefined>, options: UseWheelScrollDirectionOptions = {}) {
```

Comment:
`MaybeRef<HTMLElement | SVGElement | Window | Document | null | undefined>` can be extracted to `unrefElement`.

Reply:
Thanks.